quinn/src/platform/unix.rs

```diff
@@ -332,19 +336,55 @@ pub fn caps() -> UdpCapabilities {
 
 const CMSG_LEN: usize = 80;
 
+#[inline]
```

Comment:
I don't think this annotation is justified on internal functions (also below) unless we've shown that the optimizer gets it wrong and it matters.